src/pd/mod.rs

```diff
@@ -187,6 +187,11 @@ pub trait PdClient: Send + Sync {
     fn scatter_region(&self, _: RegionInfo) -> Result<()> {
         unimplemented!();
     }
+
+    // Register a handler to the client, it will be invoked after reconnecting to PD.
+    //
+    // Please note that this method should only be called once.
+    fn on_reconnect<F: Fn() + Sync + Send + 'static>(&self, _: F) {}
```

Comment:
Em, what I mean before is to use on_reconnect as the field name, here we should add a verb like set explicitly.